pkgs/applications/science/biology/dcm2niix/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libyamlcpp
+}:
+
+let
+  version = "v1.0.20170130";
```

Comment:
It's common practice not to have the "v" in the version number attribute but instead in the `rev` attribute passed to `fetchFromGitHub`.